my pc

abchois opened the pull request.

onebeartoe committed:
Findbugs was updated to the latest version to work with Java 8.

onebeartoe committed:
A duplicate dependency entry was removed.

onebeartoe committed:
A temporary project URL  was added.

onebeartoe committed:
The compiler plugin configuration was corrected.

onebeartoe committed:
The checkstyle version was updated.

onebeartoe committed:
The reporting plugin was defined, so that the Maven site goal (hopefully) succeeds on Cloudbees/Jenkins.

onebeartoe committed:
A JXR plugin was added.

onebeartoe committed:
The artifact ID was updated.

onebeartoe committed:
The parent POM tag was added.

onebeartoe committed:
The whitespace was formatted.

onebeartoe committed:
The parent POM tag was added.

onebeartoe committed:
An offset for the scrolling text y-axis was added, as well as tips for using scrolling text with a rectangle matrix (32 x 16 for example).

onebeartoe committed:
Changes were maded for the y-axis offset for scrolling text.

onebeartoe committed:
Changes were maded for the y-axis offset for scrolling text.

onebeartoe committed:
An example init.d daemon script was added to run the applicaion on boot.

onebeartoe committed:
Merge branch 'master' of https://github.com/alinke/PIXEL

onebeartoe committed:
An implementation for TextFileReader is now specified.

onebeartoe committed:
The import statements were corrected.

onebeartoe commented:
@abchois thanks for the pull request.  Can you provide a of description of what this pull has?

alinke committed:
added PIXLZ025 board support - this is the iOS/Android 32x32 Only firmware

alinke committed:
changed default led matrix to PIXEL V2 / Adafruit 32x32 from the older Seeed 32x32 panel

alinke committed:
changed default panel to pixel v2 / adafruit 32x32 for the pixelweb app

onebeartoe committed:
Merge branch 'master' into master

onebeartoe committed:
update the repository URL and update code for the latest onebeartoe API

onebeartoe committed:
merge conflict resolution

onebeartoe committed:
add continuous integration

onebeartoe committed:
remove the group ID and version

onebeartoe committed:
remove the group ID and version

onebeartoe committed:
update the group ID and version

onebeartoe committed:
update the group ID and version

onebeartoe committed:
update the group ID and version

alinke committed:
updated pom to java 7

alinke committed:
updated led matrix panels in ioio libs

alinke committed:
in pixel-web-enabled, changed resources directory name from "stills" to "images" as the files weren't extracting on first run

alinke committed:
renamed "stills" to "images"

alinke committed:
adding new supported led matrix to pixel commons

alinke committed:
WIP, added new handler for arcade mode, this handler will handle both gifs and still image pngs

alinke committed:
added error handling if gif or png is not there

alinke committed:
arcade handler now can take png or gif

alinke committed:
detecting the pixel port from settings.ini

alinke committed:
getting the PIXEL port from settings.ini

alinke committed:
added hack to change a single frame gif to two frames due to raspberry pi bug

alinke committed:
removed the pixel port option since we're using setttings.ini instead

alinke committed:
fixed bug where /gifsource was not getting extracted on first run, also now extracting /console, /mame/ and auto-creating the folder structure for pixelcade

alinke committed:
fixed bug where single frame gif wasn't decoding, changed base directory to pixelcade from pixel, and also removed the extra arcade sub-directory

alinke committed:
added png versions of the mame marquees

alinke committed:
fixed web-content files and add install files for Pi/Mac only

alinke committed:
deleted a few files not needed

alinke committed:
added more supported LED matrix types

alinke committed:
reading the LED panel type from settings.ini, added first install files for retropie and mac users

alinke committed:
added new led panel types including the frame types and sizes

alinke committed:
now handling both stream and writes for both PNG and GIFs (still need to add write for PNGs). If in stream mode and there is both a PNG and a GIF, take the PNG first. If in write mode and there is both a PNG and a GIF, take the GIF first. Added HyperSpin to RetroPie console mapping logic.

alinke committed:
getting the PIXEL port from settings.ini

alinke committed:
swapped out single frame GIFs with PNGs because PNGs will be faster and don't need to decode

alinke committed:
WIP of pixelcade front end that will make calls to pixelweb, this .exe can be called from various front ends. All it does is takes the console and rom from the arcade front end and then makes a REST call to pixelweb

alinke committed:
defaulting the com port to com99 so we can prompt the user if they didn't change it

alinke committed:
garbage push

alinke committed:
adding PNG write

alinke committed:
POM change for pixelcade-front-end, maven not working yet for it

alinke committed:
renamed the pixelcade front end class to PixelcadeFrontEnd from WebEnabledPixel

alinke committed:
typos

alinke committed:
removed 15s delay on GIF writes, also fixed bug where switching from write to stream was not working, the fix was add in pixel.interactive before the write

alinke committed:
pixelcade front end working now, does the http rest call

alinke committed:
garbage push

alinke committed:
changed from user.home to user.dir, this way we don't need to force the install on windows for user home and user can install wherever they want and pixelweb will take the working directory where the exe was run

alinke committed:
we won't extract the arcade and mame files for windows since the installer takes care of that already

alinke committed:
changed the delay from 15 sec to 10 ms

alinke committed:
added error handling to catch crash if pixelcade is run but pixelweb was not run prior. Prompt the user to run pixelweb first include a windows pop up. TO DO see later if we can just launch pixelweb directly which tried but didn't work

alinke committed:
may need to change home dirctory back to user home vs. working directory

alinke committed:
still crashing on windows but not on Mac  when switching from PNG stream to GIF and vice versa

alinke committed:
garbage push

alinke committed:
added a thread sleep before playlocal on writes, this solves the ioio connection lost issue on windows (issue not there on mac)

alinke committed:
still open issue when streaming a gif and then streaming another gif

alinke committed:
added graceful exit if pixelweb was already running when launched

alinke committed:
changed the home directory for Pi and Mac to user.home / pixelcade, for windows, it's still the directory that pixelweb.jar was installed in by the windows installer

alinke committed:
looks like bug is fixed where on streaming will get ioio disconnects, turns out that replacing the java timer with ScheduledExecutorService allows the timertask to shutdown gracefully. pretty sure what was happening with the java timer is that when stopping the timer, pixel was in the middle of sending a frame which caused the disconnect

alinke committed:
updated to latest version of purejavacomm

alinke committed:
garbage push

alinke committed:
switching over to the new timers

alinke committed:
release 2.0.1, cleaned up println statements

alinke committed:
added rest api to quit pixelweb

alinke committed:
added common file logger (logme) and pixelcade.exe will now auto-launch pixelweb.exe if it's not already running

alinke committed:
updated mame PNGs and GIFs

alinke committed:
version change

alinke committed:
added logging

alinke committed:
added logger just for pixelcade.exe front end which writes to pixelcade.log, pixelweb now writes to pixelweb.log

alinke committed:
added pixelcade.jar to Pi one time install, updated settings.ini

alinke committed:
added support for scrolling text including text, color, and speed

alinke committed:
changed names of logging files

alinke committed:
tweaks for scrolling text in pixel.java

alinke committed:
limiting pixelcade.log file size to 2 MB max

alinke committed:
scrolling text controller color can be a hex code (existing) or also accept a color string (red, yellow, green, etc.)

alinke committed:
added an interactivemode() on stoptimer as there was a bug where the scrolling text did not always display

alinke committed:
adding max log size

alinke committed:
these now support both PNG and GIFs and also writes

alinke committed:
changing default panel to 15 (128x32)

alinke committed:
added arcade marquees and animations, updated artwork

alinke committed:
added an interactivemode here as sometimes the text was working

alinke committed:
default panel to 15

alinke committed:
WIP adding Q Feature

alinke committed:
added more supported panels

alinke committed:
added library to parse URL parameters

alinke committed:
garbage push

alinke committed:
fixes for q feature, added timer for still images looping

alinke committed:
Q feature

alinke committed:
added 6 digit length check to verify valid hex value (no #)

alinke committed:
added more parameters including loop

alinke committed:
bug fixes for queue, mainly release 2.0.6

alinke committed:
the extra Interactivemode() in stopexistingtimer() was causing problems on windows only, specifically a scrolling text for example could not be interrupted by a png, removed and that solves that problem

alinke committed:
added backgroundmode flag which is needed to run pixelweb.jar as a service on the Pi

alinke committed:
added mapping for mame-libretro as a console which will be coming from retropie

alinke committed:
misc fixes

alinke committed:
changed default led display setting to 64x32 for RetroPie only

alinke committed:
updated Pi scripts

alinke committed:
removed one Pi file not needed

alinke committed:
added if statement so -t is not there for non MAME

alinke committed:
2.0.8 Changes:

Commands sent before Pixel Connected now go to the Q (when pixel connects, then it will run through the Q)
Added API for just console/emulator images / alt text
http://localhost:8080/console/<stream or write>/<console-name>

http://localhost:8080/console/<stream or write>/<console-name>?t=[alt-scrolling-text]

http://localhost:8080/console/stream/nes?t=hello&l=2
Added API dedicated for just the animations folder (but for animations that are game specific, use the existing arcade API). But you can use this one for animations not game specific like screen saver, etc.
http://localhost:8080/animations/<stream or write>/<animation name>

http://localhost:8080/animations/<stream or write>/<animation name>?l=x

PNG images with loops will now blank screen after loop done

Still need to work on the scrolling text speed, have an idea how to fix that, will try this weekend

note for still images / PNGs with a l=1, it doesn’t display, so use l=2 minimum for those, need to look into that

alinke committed:
timer bug fix for png stills

alinke committed:
bug fix for queue items that had a comma and tweaked scrolling text for windows

alinke committed:
added game title feature to v2.1.2, -gt flag that scrolls Araknoid from 1988 by Taito (only if there is not a matching image)

alinke committed:
updated pi script for the gt flag

alinke committed:
added scrollspeed to all commands include the Queue, added 64x64 , 64x64C, and 64x32C support

alinke committed:
added latest pixelcade.jar to retropie files

alinke committed:
added console.csv file to map the consoles

alinke committed:
added loop=99999 flag which is a new feature in the Q, if loop = 99999 then the Q is allowed to finish and the command with loop = 99999 will play indefinitely

onebeartoe committed:
add a value for dist:

alinke committed:
added support for P2.5 64x32 panel from SYRLED, green and blue lines were swapped

alinke committed:
added new art in images

alinke committed:
initial arduino release for the  max7219 matrix and 7 segment display integration

alinke committed:
bug fixes in arduino code for max7219

alinke committed:
added oled support to the display accessory, stable release

alinke committed:
Arduino - added support for multiple LED matrix displays or zones

alinke committed:
changed from waiting for IOIO connection to waiting for PIXEL connection in the console messages

alinke committed:
added support for sub display accessory, java is writing to an additional serial port that an arduino listens to. We sent game meta data from the file "allmetadata.csv" to the displays (led matrix single color, 7 segment, and mini oled) connected to the arduino

alinke committed:
garbage push

alinke committed:
added support for multiple matrix displays (matrix, 7segment, and oled with address change)

alinke committed:
for the arduino integration, had an extra % in the string and added console call

alinke committed:
fixed bug in parsing from java, added some logic for incomplete data

alinke committed:
added call to arduino display accessory to show console meta data while scrolling through consoles in the front ends

alinke committed:
added api to switch pixel local playback

alinke committed:
small tweaks

alinke committed:
bug fix: animations now working from pixelcade art browser

alinke committed:
added localplayback to the Q such that when LEDBlinky calls http://localhost:8080/localplayback?l=0 when exiting the arcade front end, the thanks for playing text will scroll and then once down, Pixelcade goes into local playback mode and plays the last image/animation that was written to the SD card

alinke committed:
added two new board firmware strings, PIXLA025 (for pixelcade 128x32) and PIXLP025 (for pixelcade pindmd)

alinke committed:
removed default artwork from pixelweb.jar to make things smaller and now the automated Pi install script will add this content so it's not needed here in the .jar file

alinke committed:
if a gif and png are both there (ex. pacman.gif and pacman.png), then will play the gif one time and then stay on the logo PNG

alinke committed:
changed default to 128x32 in settings.ini

alinke committed:
added emulationstation mod from Kai

alinke committed:
skipping installing extra files as the pi installer now covers

alinke committed:
unzipped files

alinke committed:
added the modified files for pixelcade to the readme for ES

alinke committed:
Updated with URLEncode

alinke committed:
bug fix, added a blank frame when both a gif and png is present as a new gif was not killing the previous gif

alinke committed:
bug fix - when _ or space replaces in file name , the Q name was wrong, fixed

alinke committed:
Update README.md

updated build instructions

alinke committed:
Update README.md

alinke committed:
Update README.md

added 4 files that are modified for the ES and Pixelcade integration

alinke committed:
Update README.md

alinke committed:
added new handler for uploading console images including ALU and also it works with curl

alinke committed:
new handler to upload gif and png to console dirs including ALU and also works with curl

alinke committed:
Merge branch 'master' of https://github.com/alinke/PIXEL

* 'master' of https://github.com/alinke/PIXEL:
  Update README.md
  Update README.md
  Update README.md
  Update README.md

alinke committed:
frame rates for gif decoding

alinke committed:
version upgrade and adding platform handler

alinke committed:
removed logging to reduce clutter in console output

alinke committed:
garbage push

alinke committed:
Update README.md

added compile method for Pi 4 for ES

alinke committed:
Update README.md

alinke committed:
added ?r parameter which will play a random animation

alinke committed:
Merge branch 'master' of https://github.com/alinke/PIXEL

* 'master' of https://github.com/alinke/PIXEL:
  Update README.md
  Update README.md

alinke committed:
new rest call to accept an uploaded png or gif, this is for the ALU integration

alinke committed:
lowercase in the gif decoder was causing gif decoding for long file names not to work on the Pi, still need to test though this didn't break the PC

alinke committed:
added "please wait, one time decoding" to the arcade animations if decoding, not really  needed on a fast PC but on a slower Pi, this tells the user what is going on as opposed to a blank screen

alinke committed:
added Pixelcade Factory Test Android App used by the manufacturer to test Pixelcade board before shipping

alinke committed:
cleaned up some code comments on the arduino sub-display code

alinke committed:
Pi Image and Setup Scripts

alinke committed:
API to return a random gif from the animations folder

alinke committed:
pi image instructions updated

alinke committed:
pi wifi access point

alinke committed:
garbage push

alinke committed:
merge conflict

alinke committed:
Merge branch 'master' of https://github.com/alinke/PIXEL

* 'master' of https://github.com/alinke/PIXEL:
  garbage push
  pi wifi access point
  pi image instructions updated
  API to return a random gif from the animations folder
  Pi Image and Setup Scripts
  cleaned up some code comments on the arduino sub-display code

alinke committed:
ES screen saver code tweaks

alinke committed:
screen saver crash debugging

alinke committed:
hoping this resolves screen saver crashes on pi

alinke committed:
updated Pi scripts

alinke committed:
updated scripts and es mod for pi 4

alinke committed:
updated readme with build instructions

alinke committed:
pi image and script updates

alinke committed:
added attract mode setup to setup.sh

alinke committed:
first pass at lcd marquee support

alinke committed:
path fix for windows lcd module

alinke committed:
LCD tweaks

alinke committed:
updated pi image steps

kaicherry committed:
LCD Support Windows inhancements

- Adds support for video playback (use of 1280x390 mp4 recommended and tested)
WIP: LCD text Scroller

- Various refactors and fixes

alinke committed:
Merge pull request #10 from kaicherry/master

LCD Support Windows enhnacements

alinke committed:
memory tweaks for LCD

kaicherry committed:
Fixed displayImage/WindowsLCD issue

Only make one.

alinke committed:
Merge pull request #11 from kaicherry/master

Fixed displayImage/WindowsLCD issue

alinke committed:
reverted back LCD fixes

alinke committed:
scrolling speed bug fixes, changed scrollspeed default to 0 so it gets picked up in webenabledpixel

alinke committed:
removed unused dependenacy which reduced .jar file size from 12 mb to 10 mb

alinke committed:
windows scrolling text for LCD

alinke committed:
LCD windows changes

alinke committed:
console LCD fix

alinke committed:
lcd marquee fixes

alinke committed:
LCD marquee tweaks

alinke committed:
pixelhome fix for LCD

alinke committed:
windows LCD fixes

kaicherry committed:
Fixed UUID call

Fixed UUID call

kaicherry committed:
Long text scroll fix

Long text scroll fix

alinke committed:
Merge pull request #14 from kaicherry/master

UUID + school fixes

Commit:
Adds support for anigifs on linux, adds hooks for text scroller

Commit:
Merge branch 'master' of https://github.com/alinke/PIXEL
Adds GIFs

kaicherry committed:
AniGif Support

Adds support for god via helper, adds hooks for scroller

kaicherry committed:
Tweaks and cleanup

kaicherry committed:
Scroller Hooks and clean up

kaicherry committed:
Support and fixes for Pi Scroller

This fixes (some) cases where command were not being sent to Lcdpixcade instances on pi because there were set to only run on Windows

This also fixes an issue where if a file was not available as a LCD marquee, input could find the normal one, it asks the lcd to display a marquee it doesn't have

Fixed issue where when KCD marquee was not available, the Alt text was not sent and shown

kaicherry committed:
Clean up + pi scroller

Commit:
Fixes and support for pi scroller implementation

kaicherry committed:
Modifies internal logic for scroller integration

Switches the logic from the older, basic model for locating lcd banners. resets/blanks old marquee before scrolling, some clean up

alinke committed:
Merge pull request #15 from kaicherry/master

Gifs

kaicherry committed:
Create maven.yml

alinke committed:
adding in latest changes for skrola and gifs on pi, modified arcadehttphandler for LCD calls

alinke committed:
gifs on LCD were getting called twice

Commit:
merge(?)

Commit:
font color conversion

Commit:
Merge branch 'master' of https://github.com/alinke/PIXEL

Commit:
Merge branch 'master' of https://github.com/kaicherry/PIXEL

kaicherry committed:
Merge pull request #2 from kaicherry/ref/kaicherry-ReduxT2

Ref/kaicherry redux t2

kaicherry committed:
Delete maven.yml

kaicherry committed:
Delete maven.yml

kaicherry committed:
Merge pull request #3 from kaicherry/ref/kaicherry-ReduxT2

Delete maven.yml

alinke committed:
Merge pull request #16 from kaicherry/ref/kaicherry-ReduxT2

Ref/kaicherry redux t2

kaicherry committed:
Merge pull request #4 from alinke/master

Merge pull request #16 from kaicherry/ref/kaicherry-ReduxT2

alinke committed:
Merge pull request #17 from kaicherry/master

Full sync up

kaicherry committed:
Fix for gif/scroll collision

alinke committed:
Merge pull request #18 from kaicherry/master

Fix for gif/scroll collision

alinke committed:
fixing multiple LCD calls issue

kaicherry committed:
Merge pull request #1 from alinke/master

Als master

alinke committed:
console changes for LCD

alinke committed:
bug fix: setting the color was not in the right place

alinke committed:
upgraded libraries purejavacomm to 1.0.3 and jna to 5.5

alinke committed:
WIP - works on Pi and PC, still need to clean up code

alinke committed:
code clean-up

alinke committed:
pixelhome fix

alinke committed:
kai's updates to LCD

Commit:
Updating to current Retropie Master, includes Pixelcade patches

Commit:
Added missing GL renderers...yikes!

Commit:
Re-add resourse dir installation, edited metatdata, removed build fragments

alinke committed:
Merge pull request #20 from kaicherry/master

Updates to latest pi ES, + pixelcade patches

alinke committed:
Update README.md
